### DISCUSSION TITLE
Lyrics button not showing up?

### DISCUSSION THREAD

**Discussion by GhosttRiderrr** [SpotX-Official/SpotX#451] [Question]
So i just installed spotX, but for whatever reason the lyrics button isn't there anymore. Is this an intended feature? How do i bring it back? Right now the lyrics button only shows up if i go into fullscreen which is rather annoying.
![image](https://github.com/amd64fox/SpotX/assets/137924797/30a1e8f7-22c8-49ce-a596-90af5d78b7ae)

**Comment by amd64fox** [comment]
I am also rather annoyed by people who do not know how to search for [similar](https://github.com/amd64fox/SpotX/issues/395) problems and read the documentation.
